geode-core/src/main/java/org/apache/geode/cache/query/CqQuery.java

```diff
@@ -155,4 +155,12 @@
    */
   boolean isDurable();
 
+  /**
+   * This allows to check if the CQ use option to suppress CQ update notification.
+   *
+   * @return boolean true if update is suppressed, false otherwise
+   * @since Geode 14.0
+   */
+  boolean isUpdateSuppressed();
```

Comment:
@mivanac I quite like the idea of possibly limiting what notifications one would like to receive. I think that the approach is maybe a little limiting, as it is specifically focussed on  "update" only. Could we achieve a much more encompassing solution for delete or create as well? Maybe an inclusions or exclusions list of events that the end result would like to receive?

Given that this is an external facing API, I would encourage changes to be more encompassing, in order to avoid many API changes/deprecations of external facing APIs.
@upthewaterspout @agingade @nabarunnag could anyone of you possibly be able to work with @mivanac to come up with an API that is possibly a little more future proofed.